easybuild/easyconfigs/s/scikit-lego/scikit-lego-0.6.16-foss-2022a.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'PythonBundle'
+
+name = 'scikit-lego'
+version = '0.6.16'
+
+homepage = 'https://github.com/koaning/scikit-lego'
+description = """
+We love scikit learn but very often we find ourselves writing custom transformers, metrics and
+models. The goal of this project is to attempt to consolidate these into a package that offers
+code quality/testing. 
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+github_account = 'koaning'
+
+builddependencies = [
+    ('binutils', '2.38'),
```

Comment:
`binutils` is not required when using `foss` toolchain (only when using `GCCcore` as toolchain)